main.go

```diff
@@ -143,18 +155,12 @@ func main() {
 	store.InitTables(db)
 
 	bnbExecutor := bnb.NewExecutor(config.BnbConfig.RpcAddr, types.ChainNetwork(bnbNetwork), config.BnbConfig)
+	kavaExecutor := kava.NewExecutor(config.KavaConfig.RpcAddr, client.ChainNetwork(kavaNetwork), config.KavaConfig)
```

Comment:
should not just delete eth executor,  we should make it configurable

Reply:
added in https://github.com/binance-chain/bep3-deputy/pull/1/commits/f5752cb6343bceb0f2ac20040a42ac3ab8b27265, https://github.com/binance-chain/bep3-deputy/pull/1/commits/b0d8d270aee5824a47ef48629438f78621db9a31